src/notification/email_sender.py

```diff
@@ -151,15 +151,18 @@ def generate_email_content(self) -> str:
                             if (
                                 not self.report_config.hide_filters
                                 and department != "single_department"
-                            ):
-                                blocks.append(f"**{department}**")
+                            ):                            
+                                blocks.append(f"**{department}**") 
```

Comment:
```suggestion
                                blocks.append(f"**{department}**")
```

Espaço em branco extra.